react/Header/NotificationsLink/NotificationsLinkContainer.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getCookieFromString } from './utils';
+import { AUTHENTICATED } from '../../private/authStatusTypes';
+import { EXPERIMENT_ID, AB_CRUNCH_URL } from './constants';
+import NotificationsLink from './NotificationsLink';
+
+class NotificationsLinkContainer extends React.Component {
+  static propTypes = {
+    authenticationStatus: PropTypes.bool,
+    linkRenderer: PropTypes.element
+  };
+
+  state = {
+    isInExperiment: false
+  };
+
+  componentDidMount() {
+    let visitorId = null;
+
+    try {
+      if (typeof document !== 'undefined') {
+        visitorId = getCookieFromString('JobseekerVisitorId', document.cookie);
+      }
+      if (visitorId) {
+        const url = `${AB_CRUNCH_URL}/${visitorId}`;
+
+        fetch(url)
+          .then(response => response.json())
+          .then(response => {
+            const isInExperiment = Boolean(response.experiments[EXPERIMENT_ID]);
+
+            if (isInExperiment) {
+              this.setState({ isInExperiment: true });
+            }
+          })
+          .catch(() => {});
+      }
+      // eslint-disable-next-line no-empty
+    } catch (e) {}
+  }
+
+  render() {
+    const { authenticationStatus, linkRenderer } = this.props;
```

Comment:
Just a idea: I'm liking hooks for this. But that might be the first hook in seek-style-guide requiring us to up the peer-dependency.

```js
const [isInExperiment, setIsInExperiment] = useState(false);
useEffect(() => {
  fetchIsInExperiment().then(setIsInExperiment)
}, [])
```

Perhaps even a `useIsInExperiment(visitorId, url)` hook.

Reply:
This is a good point... we will need to mark this as a breaking change if we introduce hooks here because we currently only require react 16.3. And this will slow down some teams in updating their headers

Reply:
Love the idea. Would prefer to do this at a later stage though, in favour of getting this live quicker. This feature needs a follow up PR anyway, to remove the segmentation/feature toggle call.